training-service/src/app/pages/mail-form/mail-form.component.scss

```diff
@@ -1,6 +1,19 @@
 $red-alert: #f00;
 $green-alert: green;
+.submitForm{
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  position: absolute;
```

Comment:
Please refactor position absolute to something responsive, for example flex.

Reply:
change for realtive